### DISCUSSION TITLE
MIDI remapping for Humanize Velocity and other misc properties?

### DISCUSSION THREAD

**Discussion by coachtrip** [hydrogen-music/hydrogen#1683] [Hydrogen on Linux]
Is there a way to map Humanize Velocity to a MIDI slider?

**Comment by theGreatWhiteShark** [comment]
No, not yet. I created a feature request (#1684).

**Reply by coachtrip** [reply]
Thank you so much!  I've found that the Humanize Velocity serves as a decent replacement to a 'Fill' button, adding some quick randomization whenever needed.  And being able to trigger it via MIDI would be amazing!

**Comment by elpescado** [comment]
Hi,I think I’ve implemented that as part of https://github.com/hydrogen-music/hydrogen/pull/525 That bit could be backported.Cheers,PrzemekWiadomość napisana przez coachtrip ***@***.***> w dniu 09.12.2022, o godz. 23:26:﻿
Thank you so much!  I've found that the Humanize Velocity serves as a decent replacement to a 'Fill' button, adding some quick randomization whenever needed.  And being able to trigger it via MIDI would be amazing!

**Reply by coachtrip** [reply]
Oh wow!! This looks even better than what I was looking for! 
I'm only a few months into getting up and running in Linux music-wise, and I'm having some trouble figuring out how to build this particular branch.  I've been following this link (https://github.com/hydrogen-music/hydrogen/blob/master/INSTALL.md) to build Hydrogen, where along this process should I append in order to add #525?

**Reply by theGreatWhiteShark** [reply]
```bash
$ git remote add elpescado https://github.com/elpescado/hydrogen/
$ git fetch elpescado
$ git checkout feature/probability20
$ mkdir build && cd build
$ cmake ..
$ make && sudo make install
```

**Reply by theGreatWhiteShark** [reply]
> Hi,I think I’ve implemented that as part of [#525](https://github.com/hydrogen-music/hydrogen/pull/525) That bit could be backported.

We are, again, on feature freeze for v1.2 but we could add it in v1.3 (there will be probably some work to rebase it though as there were significant changes in the GUI since your PR)

**Reply by coachtrip** [reply]
I'm getting this error when typing in `git remote add elpescado https://github.com/elpescado/hydrogen/`

```
fatal: not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set)
```

**Reply by theGreatWhiteShark** [reply]
You have to run the commands listed above in the `hydrogen` folder you cloned via `git`.

**Reply by coachtrip** [reply]
ahhh so i run these commands after 'cd hydrogen',  awesome thank you for help!

**Comment by theGreatWhiteShark** [comment]
Implemented in https://github.com/hydrogen-music/hydrogen/commit/df52183d7db743fb362eca3f60fe0676cff973b2
